Introduction (#19)
greenelab/annorxiver_manuscript: content/02.introduction.md

Common preprint repositories include arXiv [@doi:10.1108/14666180010345564], bioRxiv [@doi:10.1101/833400] and medRxiv [@url:https://www.medrxiv.org/]; however, there are over 60 different preprint repositories [@url:https://scholarlykitchen.sspnet.org/2019/10/16/the-second-wave-of-preprint-servers-how-can-publishers-keep-afloat/].
Overall, preprints are becoming widely accepted within the life sciences community [@doi:10.1371/journal.pmed.1002549; @doi:10.1038/530265a; @doi:10.1371/journal.pbio.3000151; @doi:10.1099/mgen.0.000259].

The scientific community has begun to analyze the impact of preprints in the life sciences.
Studies have found that preprints are being posted at a high rate [@doi:10.1371/journal.pbio.3000269] and are rapidly shared on social media, routinely downloaded, and cited often [@doi:10.1371/journal.pone.0047523].
Certain categories of preprints seem to be read and shared differently by both scientists and non-scientists [@doi:10.1101/2020.03.06.981589].
Across preprint servers, analyses suggest that between two thirds and three quarters of preprints are eventually published [@doi:10.7554/eLife.45133.001; @doi:10.1002/pra2.175].

> Studies have found that preprints often lower the amount of time required to be published in a journal [@doi:10.2139/ssrn.3455146].

Author: >Whatever drives the need to update preprints, there is a benefit of sorts: the time from latest available preprint to publication is over three weeks shorter than that between first preprint submission and publication (Figure 2). But when we add in time spent working on additional versions (7.7 weeks median average) — overall, preprints with just one version on bioRxiv are published fastest. 

Figure 2 shows that preprints with multiple versions take the lowest time to get published, but when they factor in the amount of time between versions it is higher. Now that you mention it a figure to show this would have been nice.

> Studies have found that preprints often lower the amount of time required to be published in a journal [@doi:10.2139/ssrn.3455146].

Reviewer: If I'm reading this correctly, it doesn't say that papers with preprints get published faster than those without though. I read the sentence as saying that. Did they do any analyses of papers that lacked preprints?

> Studies have found that preprints often lower the amount of time required to be published in a journal [@doi:10.2139/ssrn.3455146].

Reviewer: It's also possible I'm misreading what the sentence in our paper is asserting.

> Studies have found that preprints often lower the amount of time required to be published in a journal [@doi:10.2139/ssrn.3455146].

Author: I see now. I'll edit that sentence to be more clear.

Articles with preprints were more cited and more discussed than those without them [@doi:10.1162/qss_a_00043; @doi:10.7554/eLife.52646].

> This existing work largely focuses on the metadata associated with life sciences preprints.

Reviewer: ```suggestion
Existing work largely focuses on the metadata associated with life sciences preprints.

```

## Introduction

Preprints are scholarly works that have yet to undergo the peer review process.

> They were first used by physicists back in the 1990s [@doi:10.15252/embj.201695531] and have become an essential tool for the scientific community [@doi:10.1038/530265a; @doi:10.1016/j.bpj.2016.06.035; @doi:10.1126/science.aaf9133; @doi:10.1038/s41557-020-0477-5].

Reviewer: I added this below - but it might need to get moved up by one sentence.

Perfect example would be on the explosion of preprints for the COVID-19 pandemic as a great deal of untrustworthy preprints were posted in bioRxiv and medRxiv [@doi:10.1101/2020.09.04.20188771; @doi:10.1136/bmj.m3111].
Additionally, there has been skepticism on the success of preprints outlasting the publication process [@doi:10.1002/leap.1265].
Despite the pros and cons of preprints, majority of the scientific community view them in a positive light and have encouraged more and more scientists to adopt this form of communication [@doi:10.1371/journal.pmed.1002549; @doi:10.1038/530265a; @doi:10.1371/journal.pbio.3000151; @doi:10.1099/mgen.0.000259].

The bulk of analyses that involve preprints focus on areas of preprint popularity, the preprint audience, publication success of preprints and validating preprint claims.
Regarding preprint popularity, authors found that preprints have a short time delay before becoming quite popular on communicative platforms such as twitter and google scholar [10.1371/journal.pone.0047523].
Similar studies have found that preprints are being posted at a high rate [@doi:10.1007/s11192-020-03430-8; @doi:10.1371/journal.pbio.3000269].

> Meanwhile authors studying the audience that reads preprints found a political link between the non-scientific community and preprint categories [@doi:10.1101/2020.03.06.981589].

Reviewer: ```suggestion
Certain categories of preprints seem to be read and shared by differently by both scientists and non-scientists [@doi:10.1101/2020.03.06.981589].
```

> In relation to publication success, there are studies that suggest preprints are being published at a high rate [@doi:10.7554/eLife.45133.001], while others suggest an inverse trend [@doi:10.1002/leap.1265; @doi:10.1101/2020.01.28.920546].

Reviewer: ```suggestion
Across preprint servers, analyses suggest that between two thirds and three quarters of preprints are eventually published [@doi:10.7554/eLife.45133.001; @doi:10.1002/asi.23044].
```

> Lastly, there have been a number of studies that question the validity of preprint claims in respect to the COVID-19 pandemic [@doi:10.1101/2020.05.22.111294; @doi:10.1101/2020.09.04.20188771; @doi:10.1038/d41586-020-01394-6].

Reviewer: ```suggestion
```

> Despite the discoveries above, there is still a gap in understanding the language contained in preprints and how it changes as preprints undergo the peer review process.

Reviewer: ```suggestion
This existing work largely focuses on the metadata associated with life sciences preprints.
There is still a gap in understanding the language authors use in preprints and examining how language shifts between preprint and peer reviewed manuscript provides a unique opportunity to examine how the publishing process impacts the scholarly literature.
```

> In this paper, we seek to remedy this gap by conducting a linguistic analysis on the bioRxiv repository.

Reviewer: ```suggestion
We sought to understand the language landscape of preprints by performing a linguistic analysis of the _bioRxiv_ corpus.
```

> We conducted a series of analyses to determine the differences between preprints and published literature on a local and global scale.

Reviewer: ```suggestion
We also examine textual differences between preprints and published literature by comparing the entire corpus of preprints with that available in the open access PubMed Central repository as well as a restricted set of preprint-published pairs.
```

> We also developed a tool that provides the life sciences' community with possible journal endpoints for their respective preprints posted in the bioRxiv and medRxiv repositories.

Reviewer: ```suggestion
We find that neural-network derived document embeddings provide a useful space for determining the textual similarity of preprints.
Preprints' nearest neighbors in this space are likely to share an eventual publishing venue with the preprint, so we created a webserver that will display these neighbors for any preprint on _bioRxiv_ or _medRxiv_.
```

> Through this linguistic analysis we will gain better insight on the language contained in bioRxiv preprints as well as gain more insight on the textual requirements for the peer review process.

Reviewer: ```suggestion
Proximity in this space also reveals preprint-publication pairs that earlier analyses have been unable to identify.
Our linguistic analysis reveals the impact of the publishing process, introduces a method to identify similar preprints, demonstrates that the text content of preprints is related to their eventual publication venue, and provides a more complete picture of the fraction of preprint that are eventually published.
```
